test/es6/bug_issue_3076.js

```diff
@@ -0,0 +1,328 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function f0 (a, b) {
```

Comment:
Yes, it is better to just use forcedeferparse

---
In reply to: [120246993](https://github.com/Microsoft/ChakraCore/pull/3111#discussion_r120246993) [](ancestors = 120246993)